fee_calculator/apps/calculator/tests/test_calculation_02_lgfs_2016.py

```diff
@@ -17,3 +17,6 @@ class Lgfs2016CalculatorTestCase(
 
 
 Lgfs2016CalculatorTestCase.create_tests()
+
+valid_special_prep_scenarios = list(range(2, 13)) + list(range(19, 37)) + list(range(38, 43))
```

Comment:
I've taken a slightly different approach which I'll push the commit for shortly, let me know what you think?

I've defined valid_special_prep_scenarios within the Special Prep tests mixin to avoid duplicating that big dictionary in three separate files, and set logic for it to delete the entry it needs to if the scheme pk is 6 (LGFS 10)

Reply:
Yes, I think that works. I was hoping there was a way to just define the list once rather than repeat it for each set of tests, so good job on finding it 😁